IdentityCore/src/broker_operation/request/MSIDBrokerOperationRequest.h

```diff
@@ -36,10 +36,12 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, nullable) NSString *clientAppVersion;
 @property (nonatomic, nullable) NSString *clientAppName;
 @property (nonatomic) MSIDClientSDKType clientSDK;
+@property (nonatomic) BOOL isRunTime;
```

Comment:
Can we rename this property to something more general? "isRunTime" makes no sense to the BroserCore app :)

Reply:
Updated as ```clientBrokerKeyCapabilityNotSupported```